muxpi/set-up-image.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
+NEPTUNE_IMAGE="$1"
+IMAGES="$HOME/images"
+
+#The url for the latest successful nightly build.
+VARIANT_MINIMAL="core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi/$VARIANT_MINIMAL"
+
+VARIANT_NEPTUNE="core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/$VARIANT_NEPTUNE"
+
+VARIANT=""
+URL=""
+if [ "$NEPTUNE_IMAGE" == neptune ]; then
+   VARIANT=$VARIANT_NEPTUNE
+   URL=$URL_NEPTUNE
+else 
+   VARIANT=$VARIANT_MINIMAL
+   URL=$URL_MINIMAL
+fi
+
+mkdir -p $IMAGES
+
+wget $URL -P $IMAGES
+if [ $? == 0 ]; then
+   echo "Image downloaded"
+fi
+
+#map.json requires the image names and the number of the partition on SD-card.
+#In our case, we dont need partition numbers, we want the image to be flashed
+#on /dev/sda directly.
+echo "{\"${VARIANT}\":\"\"}" > $HOME/map.json
+echo "Json map is ready. Compressing the downloaded image... "
+   
+cd $IMAGES
+tar -czf $VARIANT.tar.gz $VARIANT
+if [ $? == 0 ]; then
+   echo "Image has finished compressing" 
+fi
+# fota requires the card device to be flashed, the json map which contains
```

Comment:
you either add spaces after # or not (please do!)

Reply:
Added